app/src/main/java/org/simple/clinic/bloodsugar/unitselection/BloodSugarUnitSelectionInit.kt

```diff
@@ -0,0 +1,11 @@
+package org.simple.clinic.bloodsugar.unitselection
+
+import com.spotify.mobius.First
+import com.spotify.mobius.Init
+
+class BloodSugarUnitSelectionInit : Init<BloodSugarUnitSelectionModel, BloodSugarUnitSelectionEffect> {
+  override fun init(model: BloodSugarUnitSelectionModel): First<BloodSugarUnitSelectionModel, BloodSugarUnitSelectionEffect> {
+    val effect = mutableSetOf(PreFillBloodSugarUnitSelected(model.bloodSugarUnitPreference))
+    return First.first(model, effect as Set<BloodSugarUnitSelectionEffect>)
```

Comment:
If we have only a single effect, we do not need to create a set of effects since we have our function `MobiusFoo#first` which accepts `vararg` parameters.